dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml

```diff
@@ -1,7 +1,8 @@
 .by_day
   %h3
     Percentages by Day
-  #chart_by_day
+  %div
+    #chart_by_day
```

Comment:
what was the motivation to put `#chart_by_day` inside a container in the first place?